app/src/pages/inside/profilePage/localizationBlock/localizationBlock.jsx

```diff
@@ -0,0 +1,116 @@
+import { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { injectIntl, intlShape, defineMessages } from 'react-intl';
+import { setStorageItem } from 'common/utils';
+import Parser from 'html-react-parser';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { langSelector, setLangAction } from 'controllers/lang';
+import { InputDropdown } from 'components/inputs/inputDropdown';
+import styles from './localizationBlock.scss';
+import EnglishFlagIcon from './img/en-flag-inline.svg';
+import RussianFlagIcon from './img/ru-flag-inline.svg';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  label: {
+    id: 'LocalizationBlock.label',
+    defaultMessage: 'Language',
+  },
+  russian: {
+    id: 'LocalizationBlock.russian',
+    defaultMessage: 'Russian',
+  },
+  english: {
+    id: 'LocalizationBlock.english',
+    defaultMessage: 'English (United States)',
+  },
+  note: {
+    id: 'LocalizationBlock.note',
+    defaultMessage: 'Note',
+  },
+  reload: {
+    id: 'LocalizationBlock.reload',
+    defaultMessage: `You have to <a>Reload</a> the page to implement language change.`,
+  },
+  contribute: {
+    id: 'LocalizationBlock.contribute',
+    defaultMessage: `Ru lang in beta. Please help us to translate it, send your PR to this <a target='_blank' href='https://github.com/reportportal/service-ui/blob/develop/src/main/resources/public/js/src/localizations/ru-RU.js' >file.</a>`,
+  },
+});
+
+@connect(
+  (state) => ({
+    lang: langSelector(state),
+  }),
+  {
+    setLangAction,
+  },
+)
+@injectIntl
+export class LocalizationBlock extends PureComponent {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    lang: PropTypes.string.isRequired,
+    setLangAction: PropTypes.func.isRequired,
+  };
+
+  state = {
+    lang: this.props.lang,
```

Comment:
Because we change language not after click in dropdown option, but after click on 'Reload' in note section. Between theese two events we need to save select lang in state.